packages/cra-template/template/LICENSE

```diff
@@ -0,0 +1,21 @@
+    MIT License
```

Comment:
nit, is this supposed to be indented?

Reply:
dunno, looks like we don't run prettier on these kind of files (because no extension)?

Reply:
Yeah I'm not sure what prettier parser/formatter would be used for a license file. I was just surprised to see an indent somehow get in there since that doesn't usually happen.

Reply:
No probs, that's a simple fix; lemme apply it!